Ginger/GingerTest/POMs/JavaPOM.cs

```diff
@@ -0,0 +1,404 @@
+
+            ElementInfo JListEI = mLearnedPOM.MappedUIElements.Where(x => x.ElementTypeEnum == eElementType.List).FirstOrDefault();
+
+            //Assert
+            Assert.IsTrue(ComboBoxEI.OptionalValuesObjectsList.Count > 0, "POM.PossibleValues check");
+            Assert.IsTrue(JListEI.OptionalValuesObjectsList.Count > 0, "POM.PossibleValues check");
+        }
+
+        [TestMethod]
+        public void TestPOMSetValue()
+        {
+            //Arrange
+            ElementInfo TextBoxEI = mLearnedPOM.MappedUIElements.Where(x => x.ElementTypeEnum == eElementType.TextBox).FirstOrDefault();
+
+            ActUIElement actUIElement = new ActUIElement();
+            actUIElement.ElementLocateBy = eLocateBy.POMElement;
+            actUIElement.ElementType = eElementType.TextBox;
+            actUIElement.ElementAction = ActUIElement.eElementAction.SetValue;
+            actUIElement.ElementLocateValue = mLearnedPOM.Guid + "_" + TextBoxEI.Guid;
+            actUIElement.Value = "This is Text Area";
+            actUIElement.Active = true;
+
+            //Act
+            mGR.RunAction(actUIElement);
+
+            //Assert
+            Assert.AreEqual(actUIElement.Status, Amdocs.Ginger.CoreNET.Execution.eRunStatus.Passed);
+        }
+
+        private static void StartAgent()
+        {
+            if (mGR == null)
+            {
+
+                mGR = new GingerRunner();
+                mGR.CurrentSolution = new Ginger.SolutionGeneral.Solution();
+                mBF = new BusinessFlow();
+                mBF.Activities = new ObservableList<Activity>();
+                mBF.Name = "BF Test Java Driver";
+                Platform p = new Platform();
+                p.PlatformType = ePlatformType.Java;
+                mBF.TargetApplications.Add(new TargetApplication() { AppName = "JavaTestApp" });
+                Activity activity = new Activity();
+                activity.TargetApplication = "JavaTestApp";
+                mBF.Activities.Add(activity);
+                mBF.CurrentActivity = activity;
+
+                ActLaunchJavaWSApplication LJA = new ActLaunchJavaWSApplication();
+                LJA.LaunchJavaApplication = true;
+                LJA.LaunchWithAgent = true;
+                LJA.WaitForWindowTitle = "Java Swing";
+                LJA.AddOrUpdateInputParamValue(ActLaunchJavaWSApplication.Fields.PortConfigParam, ActLaunchJavaWSApplication.ePortConfigType.Manual.ToString());
+                LJA.Port = "9898";
+                LJA.URL = TestResources.GetTestResourcesFile(@"JavaTestApp\JavaTestApp.jar");
+                activity.Acts.Add(LJA);
+                mGR.PrepActionValueExpression(LJA);
+                LJA.Execute();
+                // TODO: add wait till action done and check status
+                //if (!string.IsNullOrEmpty(LJA.Error))
+                //{
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4423505)